packages/client/src/consts/environments.ts

```diff
@@ -1,13 +1,45 @@
-export class Environment {
-  constructor(private url: string) {}
+/**
+ * @internal
+ * @privateRemarks DO NOT export constructor, just the type and variables defined with it
+ */
+class Environment {
+  constructor(public readonly name: string, private url: string) {}
 
   get gqlEndpoint() {
     return this.url;
   }
 }
 
-export const polygon = new Environment('https://api.lens.dev');
-export const mumbai = new Environment('https://api-mumbai.lens.dev');
+export type { Environment };
 
-// not exposed from the package, used in tests
-export const mumbaiSandbox = new Environment('https://api-sandbox-mumbai.lens.dev');
+export const production = new Environment('production', 'https://api.lens.dev');
+
+export const development = new Environment('development', 'https://api-mumbai.lens.dev');
```

Comment:
I'd always suggest not to hardcode the values, better if we can have an exported Environment enum and consume it everywhere.

WDYT?